eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -338,4 +338,30 @@
     <Field name="~(MAX_DURATION|NULL_TXN)"/>
     <Bug pattern="MS_SHOULD_BE_FINAL"/>
   </Match>
+
+  <!--  The return value is ignored for purpose-->
+  <Match>
+    <Class name="class com.microsoft.azure.servicebus.primitives.AsyncUtil"/>
```

Comment:
Exceptional return value of java.util.concurrent.ExecutorService.submit(Callable) ignored in com.microsoft.azure.servicebus.primitives.AsyncUtil.completeFutureExceptionally(CompletableFuture, Throwable)